tests/framework/test_time.py

```diff
@@ -512,7 +512,7 @@ def test_move_simulants_to_end(base_config: LayeredConfigTree) -> None:
     assert step_modifier_component.ts_pipeline_value.index.equals(full_pop_index)
     assert np.all(
         sim._clock.simulant_next_event_times(evens)
-        == sim._clock.stop_time + sim._clock.minimum_step_size
+        == sim._clock.stop_time + sim._clock.minimum_step_size  # type: ignore [operator]
```

Comment:
I just don't want to think about this here - not really sure the best way to handle the different types in stop time vs step_size. Happy to look closer if people want though

Reply:
We ignore this everywhere else it comes up iirc